specifyweb/frontend/js_src/lib/components/Merging/Status.tsx

```diff
@@ -67,10 +78,29 @@ export function Status({
 
   const loading = React.useContext(LoadingContext);
 
+  const percentage = Math.round((state.current / state.total) * 100);
+
   return (
     <Dialog
       buttons={
-        state.status === 'MERGING' ? (
+        state.status === 'FAILED' ? (
+          <>
+            <Button.Info
+              onClick={(): void =>
+                void downloadFile(
+                  `Merging ${mergingId} Crash Report - ${new Date().toJSON()}.txt`,
+                  responseError
```

Comment:
(optional) replace `responseError` with `produceStackTrace(responseError)` so that the file you would receive from the user contains much more useful information (including, which records the user was trying to merge)